README.md

```diff
@@ -40,7 +40,7 @@ Symfony Bundle to log Request/Response with Monolog.
 m6_web_log_bridge:
     resources:
         - %kernel.root_dir%/config/m6web_log_bridge.yml
-    content_formatter: m6web_log_bridge.log_content_provider # Provider service name
+    content_formatter: m6web_log_bridge.log_content_formatter # Provider service name
```

Comment:
I simply update the documentation since I could not find any reference to `log_content_provide`.